packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -116,6 +133,9 @@ contract Attestations is IAttestations, Ownable, Initializable, UsingRegistry, R
   mapping(bytes32 => IdentifierState) identifiers;
   mapping(address => Account) accounts;
 
+  // Mapping of authorizedAttestor addresses to account addresses
+  mapping(address => address) authorizedAttestors;
```

Comment:
Consider `authorizedBy` for consistency with https://github.com/celo-org/celo-monorepo/pull/1177/files